lib/pages/editor-page.js

```diff
@@ -152,4 +152,14 @@ export default class EditorPage extends BaseContainer {
 		const emailVerificationSelector = by.css( '.editor-ground-control__email-verification-notice' );
 		return driverHelper.isElementPresent( this.driver, emailVerificationSelector );
 	}
+
+	postSaved() {
+		const driver = this.driver;
+		const savedStatusSelector = by.css( '.editor-ground-control__save-status' );
+		return driver.wait( until.stalenessOf( driver.findElement( savedStatusSelector ) ), this.explicitWaitMS, 'The post should started to save by now' ).then( () => {
+			return driver.wait( until.elementLocated( savedStatusSelector ), this.explicitWaitMS, 'The post status should be saving by now' ).then( () => {
+				return driver.wait( until.elementTextIs( driver.findElement( savedStatusSelector ), 'Saved' ), this.explicitWaitMS, 'Post should have been saved by now' );
```

Comment:
Can we trigger this off of a data attribute on the `savedStatusSelector` element rather than the text?  Looking for "Saved" won't work if we run these tests in another locale/language.

If there's no existing class name or attribute we can use we'll need to submit a PR to wp-calypso add one (like https://github.com/Automattic/wp-calypso/pull/15894 for example)

Reply:
I put in a PR for wp-calypso to add it as an attribute [Automattic/wp-calypso#17603](https://github.com/Automattic/wp-calypso/pull/17603)